crates/sozo/src/ops/migration/mod.rs

```diff
@@ -108,90 +108,94 @@ async fn execute_strategy<P, S>(
     strategy: &mut MigrationStrategy,
     migrator: SingleOwnerAccount<P, S>,
     ws_config: &Config,
-) -> Result<MigrationOutput>
```

Comment:
`MigrationOutput` was initially planned so that we can dump all the migration details to a file. This way if user forgot to copy the world address from the CLI, they can still have access to the addresses.

Reply:
Ah got it. It added some extra complexity in plumbing things through. Perhaps we can loop back and add that feature. I'm working on #478 so we can repeatedly run `sozo migrate`. Right now, we have to spin up a new katana everytime we make a contract change